scripts/globals/spells/trust/Excenmille.lua

```diff
@@ -0,0 +1,20 @@
+-----------------------------------------
+-- Spell: Excenmille
+-----------------------------------------
+-- require("scripts/globals/trust");
+-----------------------------------------
+
+function onMagicCastingCheck(caster,target,spell)
+    if (caster:hasHate()) then
+	    caster:messageBasic(700,0,0)
+    else
+        return 0
+    end
+end
+
+function onSpellCast(caster,target,spell)
+    -- Todo: put Trusts's spawn in dialog here or in entity script?
```

Comment:
Probably in the entity script, because the spawn-in dialogue isn't the _only_ dialogue they have, and it'd be nice to keep it all in the same place.